torch/quantization/ns/mappings.py

```diff
@@ -32,39 +36,140 @@
     toq.conv3d,
     toq.conv3d_relu,
     toq.cat,
+    toq.elu,
+    toq.hardswish,
+    toq.instance_norm,
+    toq.layer_norm,
+    toq.leaky_relu,
 ])
 
 FUNS_IO_TYPE_FP32_OR_INT8 = set([
     F.relu,
```

Comment:
ideally we can generate these from quantization_patterns.py as well

Reply:
yes.  I think with the current code in quantization_patterns.py it may be confusing.  In a future PR we could move a mapping of ops to i/o dtype, supported quantization dtypes, etc, into a separate shared file, and have quantization and NS both import that.  Saving that for a future time.